packages/blockprotocol/cli/commands/publish.js

```diff
@@ -0,0 +1,185 @@
+import chalk from "chalk";
+import fs from "fs-extra";
+import path from "node:path";
+import slugify from "slugify";
+import tar from "tar";
+import tmp from "tmp-promise";
+
+import { printSpacer } from "../print-spacer.js";
+import { doesUserAgree } from "./publish/does-user-agree.js";
+import { findApiKey } from "./publish/find-api-key.js";
+import { findBlockFolder } from "./publish/find-block-folder.js";
+import { postPublishForm } from "./publish/post-form.js";
+
+// ********************* MANUAL ********************* //
+
+const manual = [
```

Comment:
Can we add typings via `/** comments */`?

Reply:
Hmmm can we use a typing from `@types/command-line-usage` instead?

Reply:
Let's just convert this to TS later if we want to have proper type security, rather than a halfway house.

Reply:
Hmm not sure we need to convert CLI code to TS — JS can be covered by TSC perfectly, with typings living in JSDoc. Not a blocker for MVP though.